cmd/generate/dockerfile-templates/DefaultDockerfile.template

```diff
@@ -3,9 +3,12 @@ ARG GO_BUILDER={{.builder}}
 ARG GO_RUNTIME=registry.access.redhat.com/ubi8/ubi-minimal
 
 FROM $GO_BUILDER as builder
-
+{{if .workdir}}
+WORKDIR /go/src/{{.main}}
 COPY . .
-
+{{else}}
+COPY . .
+{{end}}
```

Comment:
why not having it for all, without flags?

Reply:
so that we start from a clean working directory and less potential issues?

Reply:
OK, let me update the PR then.